src/Compiler/Service/SemanticClassification.fs

```diff
@@ -58,7 +58,6 @@ type SemanticClassificationType =
     | TypeDef = 35
     | Plaintext = 36
 
-[<RequireQualifiedAccess>]
```

Comment:
`RequireQualifiedAccess` is defined as AttributeTargets.Class so it should be used here

Reply:
@edgarfgp we do need the RQA here, it is one of the way to keep sanity when working in the compiler codebase.
I think with this bug you found the more appropriate action is to extend RQA to also allow structs.

Also, struct-based DUs are the same story - we definitely want to keep allowing RQA.